modules/tinymce/src/plugins/table/test/ts/browser/ui/TableCellBackgroundColorTest.ts

```diff
@@ -0,0 +1,48 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyHooks } from '@ephox/mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+import { pAssertStyleCanBeToggledOnAndOffWithoutCheckmarks } from '../../module/test/TableModifiersTestUtils';
+
+describe('browser.tinymce.plugins.table.ui.TableCellBackgroundColorTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'table',
+    indent: false,
+    toolbar: 'tablecellbackgroundcolor',
+    base_url: '/project/tinymce/js/tinymce',
+    menu: {
+      table: { title: 'Table', items: 'tablecellbackgroundcolor' },
+    },
+    menubar: 'table',
+    table_cell_background_color_map: [
+      {
+        title: 'Color',
+        value: '#ffffff',
+      }
+    ],
+  }, [ Plugin, Theme ], true);
+
+  it('TINY-7476: The color should be changed', async () => {
```

Comment:
```suggestion
  it('TINY-7476: The color should be changed for a single cell', async () => {
```
This just helps to clarify the difference between this and the next test.